core/src/test/java/com/klaytn/caver/legacy/feature/CustomGasTest.java

```diff
@@ -50,8 +52,9 @@ public class CustomGasTest {
 
     @BeforeClass
     public static void setup() throws Exception {
-        CHAIN_ID = ChainId.BAOBAB_TESTNET;
-        caver = Caver.build("https://api.baobab.klaytn.net:8651/");
+        caver = Caver.build("http://localhost:8551");
+        chainId = caver.rpc.klay.getChainID().send().getValue();
```

Comment:
Is this used?

Reply:
Oops.. chainId is not used. I'll remove it.